sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/util/StringUtils.scala

```diff
@@ -38,7 +38,7 @@ object StringUtils extends Logging {
    * throw an [[AnalysisException]].
    *
    * @param pattern the SQL pattern to convert
-   * @param escapeChar the escape string contains one character.
+   * @param escapeChar the escape character.
```

Comment:
This isn't an exposed API and rather a nit. Usually such followups or PRs are discouraged.

Reply:
OK.

Reply:
Yeah "the escape character as a one-character string" would be a little more accurate but w/e